### PR TITLE
Add video number

### PR DESCRIPTION
**Pull request by jwidness**
Add video number to the official video list.  This number is from the memory card and indicates the order of the videos on that card.

### CODE REVIEW

**Comment by jwidness** [comment]
Since I can't run the code, I can't verify it's bug-free -- could you double check that it's not accidentally displaying the full filename or anything silly?
Thanks!

**Comment by adammcmaster** [comment]
@jwidness Just to let you know, I don't think I'll get to this this week so it won't be live until after Christmas/New Year.

File: bin/generate-lists.rb [hunk 1]
```diff
@@ -80,6 +80,7 @@ def add_to_hash(db, hash, document, species)
     zooniverse_id: document['zooniverse_id'],
     tags: tags_for(db, document['zooniverse_id']).collect{ |tag| tag['_id'] },
     file: document['metadata']['file'],
+    video_num: document['metadata']['file'][-8..-5]
```
Comment (adammcmaster) [inline bin/generate-lists.rb]: @jwidness Looks like this is missing a comma at the end of the line.

**Review by adammcmaster — CHANGES_REQUESTED** [review]

**Comment by jwidness** [comment]
Oops!  Thanks for working on this -- no worries about the holidays, I was out too : )

**Review by adammcmaster — APPROVED** [review]

**Comment by adammcmaster** [comment]
Cool. We'll let this run over the weekend and double check the results next week.

**Comment by jwidness** [comment]
Great, thanks!

**Comment by adammcmaster** [comment]
@jwidness Just to say, the video numbers are on there now.

**Comment by jwidness** [comment]
Super!

**Pull request merged**